### DISCUSSION TITLE
AI-AC1-3.0

### DISCUSSION THREAD

**Discussion by robje2505** [nathanmarlor/foxess_modbus#596] [Support]
Hi all,

I own a foxess AIO-AC1-3.0 system. When I try to connect it to the integration I get the error message: 

"Error connecting to your adapter or inverter. Ensure the adapter is properly configured and is correctly wired to your inverter (see the setup link above), then try again. Details: Inverter model not supported: 'AIO-AC1-3.0'"

It's connected through a waveshare rs485 to eth poe.

Can you please make my model supported as well? 

Thanks in advance!
Robin

**Comment by canton7** [comment]
Added, assuming it's the same as a normal H1. Give v1.11.0 a try.
